content/partials/basic.html

```diff
@@ -107,3 +107,38 @@ <h3 class="ui {{ size }} header">
     <img class="ui fluid rounded image" src="{{ image }}" {%- if alt %} alt="{{ alt }}"{% endif -%} />
   </div>
 {% endmacro %}
+
+{#
+  Comparison card with logo and description
+
+  Used for platform comparison pages to show different services that can be compared.
+
+  :param url: URL to the comparison page
+  :param logo_url: URL to the platform logo
+  :param logo_alt: Alt text for the logo
+  :param title: Card title/platform name
+  :param meta: Short description below title
+  :param description: Main card description
+#}
+{% macro comparison_card(url, logo_url, logo_alt, title, meta, description) %}
+  <a class="card" href="{{ url }}">
+    <div class="content">
+      <div class="ui basic padded segment" style="height: 100px; display: flex; align-items: center; justify-content: center;">
+        <div class="ui small centered image">
+          <img src="{{ logo_url }}" alt="{{ logo_alt }}" style="max-height: 50px; width: auto;">
```

Comment:
You should instead edit the images to get similar proportions and alignment.

`.ui.small.image` already controls the size and centering of the image, inline styles or a statically sized segment shouldn't be used to dictate this.